File: app/src/main/java/com/sygic/demo3d/fragment/SygicNaviFragment.java
package com.sygic.demo3d.fragment;

import com.sygic.aura.embedded.SygicFragmentSupportV4;
import com.sygic.demo3d.SygicNaviCallback;


public class SygicNaviFragment extends SygicFragmentSupportV4 {

    @Override
    public void onResume() {
        startNavi();
        setCallback(new SygicNaviCallback(getActivity()));
        super.onResume();
    }
}

File: app/src/main/java/com/sygic/demo3d/SygicNaviFragment.kt
package com.sygic.demo3d

import com.sygic.aura.embedded.SygicFragmentSupportV4

class SygicNaviFragment : SygicFragmentSupportV4() {

    override fun onResume() {
        startNavi()
        setCallback(SygicNaviCallback(activity))
        super.onResume()
    }

}